classes/Http/Controller/Admin/SpeakersController.php

```diff
@@ -157,6 +159,9 @@ public function deleteAction(Request $req)
 
         $mapper = $spot->mapper(\OpenCFP\Domain\Entity\User::class);
         $speaker = $mapper->get($req->get('id'));
+
+        $this->removeSpeakerTalks($speaker);
+
         $response = $mapper->delete($speaker);
```

Comment:
I'm not familiar with the tools OpenCFP uses, but I think it would be good to wrap these lines in a transaction block that will do a rollback if there are any problems at all (either removing talks or deleting the speaker). This way, if the speaker deletion process fails for any reason, then all their talks won't be deleted.

Then again, maybe the Spot\Mapper tools are already doing this for you?